### DISCUSSION TITLE
Cross platform development

### DISCUSSION THREAD

**Discussion by YangUTM02** [browserengineering/book#1116] [General]
Hi all, I am not familiar with linux plateform, may need some guidance on how to:
1. How to perform the browser engineering on linux platform?
2. How to perform the browser engineering on a compiler like Visual
Studio, NetBeans and Eclipse?
3. Is that any step by step hands-on guidance on "Web Browser Engineering"
(https://browser.engineering/) chapter by chapter for me to have a
clearer picture?

**Comment by pavpanchekha** [comment]
I’m not sure what kind of guidance you’re looking for, but the WBE book and browser work fine on Windows. Visual Studio has Python support so should be fine to use, and you can install Python itself from the Microsoft Store.

**Reply by YangUTM02** [reply]
Thanks for your reply. 

When I first read the page I didn't realize that it has an instruction at the bottom at the git there, so I am asking on this.

Furthermore with my previous experience on Linux which have so many version (Ubuntu, Fedore and so on), I would have a nightmare when I getting know that it was build on Linux platform which there are lot of version change on the library that installed on Linux system that may cause me used up to few hours/days to debug on a small bug. Another reason I would asking is because I more use to Windows system where if I want to try Linux system I need to buy a new laptop ,partition my hardisk_ or the most simple is install VMware for using the Linux system. 

That is why I wondering is there any development tool that are supporting the development of web browser development that are enable the cross over platform between Linux and Windows.

I am glad and appreciate that you reply on this message. May need some guidance in future.

**Reply by pavpanchekha** [reply]
I'm not sure why you're asking about cross-platform development. If you mostly use and like Windows, just use Windows. You do not need to use Linux to read WBE and develop the source code.
